CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ## Changed
 - [[#234](https://github.com/rust-vmm/kvm-ioctls/issues/234)] vcpu: export
 reg_size as a public method.
+- [[#239](https://github.com/rust-vmm/kvm-ioctls/pull/239)] cap: add KvmCapStealTime
+capability.
```

Comment:
nit: In 0.15.0 we phrase capability addition as "Add `Cap::KvmCapStealTime` capability". Could we keep these consistent? :o

Reply:
Ok, thanks. I will fix later.